hack/e2e.sh

```diff
@@ -101,20 +101,29 @@ function kubectl_get_jsonpath() {
   fi
 }
 
+function claim_ip() {
+  IPCLAIM_NAME="$1"
+  export IPCLAIM_NAME
+  envsubst < "${REPO_ROOT}/hack/ipclaim-template.yaml" | kubectl --kubeconfig="${KUBECONFIG}" create -f - 1>&2
+  IPADDRESS_NAME=$(kubectl_get_jsonpath ipclaim "${IPCLAIM_NAME}" '{@.status.address.name}')
+  kubectl --kubeconfig="${KUBECONFIG}" get "$(append_api_group ipaddresses)" "${IPADDRESS_NAME}" -o=jsonpath='{@.spec.address}'
```

Comment:
WDYT about getting rid of append_api_group? It just seems to obfuscate what we do